libs/types-common/src/Wire/Timeout.hs

```diff
@@ -0,0 +1,32 @@
+module Wire.Timeout
```

Comment:
This module adds `Wire.*` to this package, which is a bit messy.  Add it to `Util` instead?